specification/covid-medical-exemptions.yaml

```diff
@@ -136,9 +136,15 @@ info:
     For more details see [integration testing with our RESTful APIs](https://digital.nhs.uk/developer/guides-and-documentation/testing#integration-testing-with-our-restful-apis).
 
     ## Onboarding
-    You need to get your software approved by us before it can go live with this API. We call this onboarding. The onboarding process can sometimes be quite long, so it’s worth planning well ahead.
 
-    Details to follow.
+    1. Complete an application through https://portal.developer.nhs.uk/ to the developer environment
```

Comment:
Which 1 is first?

Reply:
It's markdown.  You can use `1.` for each line and it'll convert it.

Reply:
<img width="855" alt="image" src="https://user-images.githubusercontent.com/149791/156386644-32be38fa-757d-4aaf-8b46-29307ee34a52.png">

Reply:
I would say that I like this feature because it allows you to make mistakes, but I dislike this feature because it allows you to make mistakes. Also makes it a bit less readable if you're ever looking at the non-pretty end of the markdown

Reply:
I do it because it means that if I have to re-order the file or inject a new line it doesn't cause me to have a diff across the entire block.

Reply:
In Jira they use the # character for ordered lists.